### ISSUE TITLE
Error in /onboarding/seed-ledger

### ISSUE THREAD

**Issue by rajivshah3** [iotaledger/trinity-wallet#1789]
## Error in Trinity Desktop

**Error** in **/onboarding/seed-ledger**
cannot open device with path 

[View on Bugsnag](https://app.bugsnag.com/iota-foundation/trinity-desktop/errors/5d042c5b79f6290019a29d22?event_id=5d0685800041f1a065020000&i=gh&m=ci)

## Stacktrace

    /Applications/Trinity.app/Contents/Resources/app.asar/node_modules/node-hid/nodehid.js:49 - new HID

[View full stacktrace](https://app.bugsnag.com/iota-foundation/trinity-desktop/errors/5d042c5b79f6290019a29d22?event_id=5d0685800041f1a065020000&i=gh&m=ci)

*Created by Rajiv Shah via Bugsnag*

**Comment by rajivshah3** [comment]
This issue might be related to Hardened Runtime, however there are no messages in the console from services such as `com.apple.tcc` that would indicate that Hardened Runtime is the cause. When setting up an existing Ledger account, the log messages in Console (search `Trinity`) appear to be normal. This issue does not seem to appear in dev mode

**Comment by rajivshah3** [comment]
Fixed in #1793

**Comment by rihardsgravis** [comment]
Temporary fixed in #1793

**Comment by rajivshah3** [comment]
An error linked to this issue has been marked as fixed in Bugsnag
[**Error** in **/onboarding/seed-ledger**](https://app.bugsnag.com/iota-foundation/trinity-desktop/errors/5d042c5b79f6290019a29d22?event_id=5d07eab50042165faf4f0000&i=gh&m=fx)

**Comment by rajivshah3** [comment]
An error linked to this issue has been reopened in Bugsnag
[**Error** in **/onboarding/seed-ledger**](https://app.bugsnag.com/iota-foundation/trinity-desktop/errors/5d042c5b79f6290019a29d22?event_id=5d08e067004216c9bac60000&i=gh&m=ra)

**Comment by rihardsgravis** [comment]
> An error linked to this issue has been reopened in Bugsnag
> [**Error** in **/onboarding/seed-ledger**](https://app.bugsnag.com/iota-foundation/trinity-desktop/errors/5d042c5b79f6290019a29d22?event_id=5d08e067004216c9bac60000&i=gh&m=ra)

This has a bit different error thought (no path in `0.6.0` vs exact path in `0.6.1`) which could just mean an incorrect usb read (could be temporary and fine with the next read).

**Comment by rajivshah3** [comment]
@rihardsgravis yeah, this issue was automatically reopened by Bugsnag but I guess it grouped it with the older error. The new error looks like this:

```
cannot open device with path IOService:/AppleACPIPlatformExpert/PCI0@0/AppleACPIPCI/<other stuff>
```

Could this have been caused by a `node-usb` update?
